core/src/main/java/edu/stanford/futuredata/macrobase/pipeline/CubePipeline.java

```diff
@@ -157,6 +158,14 @@ private CubeClassifier getClassifier() throws MacrobaseException {
                 classifier.setIncludeLow(includeLo);
                 return classifier;
             }
+            case "predicate": {
+                // default values for PredicateClassifier:
+                // {predicate: "==", value: 1.0}
+                final String metric = conf.get("metric");
+                final String predicateStr = conf.get("predicate", "==").trim();
```

Comment:
Same here I think it's easier to track if we do all conf loading up front in one spot and then only use the loaded instance variables in the body of the class